java-client/example-utilities/src/main/java/io/deephaven/client/examples/Ticket.java

```diff
@@ -0,0 +1,34 @@
+package io.deephaven.client.examples;
+
+import io.deephaven.client.impl.HasTicketId;
+import io.deephaven.client.impl.TicketId;
+import picocli.CommandLine.ArgGroup;
+
+public class Ticket implements HasTicketId {
+    @ArgGroup(exclusive = false)
+    ScopeField scopeField;
+
+    @ArgGroup(exclusive = false)
+    ApplicationField applicationField;
+
+    @ArgGroup(exclusive = false)
+    RawTicket rawTicket;
+
+    private HasTicketId get() {
```

Comment:
personal preference, a private method that is only called once and only saves a tiny bit of repetition isn't usually worth it - this could be easily inlined with no effect on readability.